packages/mermaid/src/diagrams/error/errorDiagram.ts

```diff
@@ -1,23 +1,13 @@
-import { DiagramDefinition } from '../../diagram-api/types.js';
-import styles from './styles.js';
-import renderer from './errorRenderer.js';
+import type { DiagramDefinition } from '../../diagram-api/types.js';
+import { renderer } from './errorRenderer.js';
+
 export const diagram: DiagramDefinition = {
-  db: {
-    clear: () => {
-      // Quite ok, clear needs to be there for error to work as a regular diagram
-    },
-  },
-  styles,
+  db: {},
   renderer,
   parser: {
     parser: { yy: {} },
-    parse: () => {
-      // no op
+    parse: (): void => {
+      return;
     },
   },
-  init: () => {
-    // no op
-  },
 };
-
-export default diagram;
```

Comment:
I think this is one place where the default exports are good. 
Writing `{ diagram as errorDiagram }` for every imported diagram doesn't look good. 

Or, we could rename the constant to errorDiagram.

Reply:
I think default export is better.
Fixed here 9ab048c7e9e193080cf00aba531863a9cae92ae7